src/systemd/sd-bus-vtable.h

```diff
@@ -185,7 +188,11 @@ struct sd_bus_vtable {
         {                                                               \
                 .type = _SD_BUS_VTABLE_END,                             \
                 .flags = 0,                                             \
-                .x = { { 0 } },                                         \
+                .x = {                                                  \
+                    .end = {                                            \
+                        .reserved = 0,                                  \
+                    },                                                  \
```

Comment:
indentaion is off. please stick to multiples of 8 chars

Reply:
hmm, weird. i see now that this is all weird in the whole file. what happened there?

Reply:
I was about to say, I simply followed the other cases